src/raftstore/store/transport.rs

```diff
@@ -17,5 +17,5 @@ use raftstore::Result;
 
 // Transports message between different raft peers.
 pub trait Transport: Send + Sync {
-    fn send(&self, msg: RaftMessage) -> Result<()>;
+    fn send(&mut self, msg: RaftMessage) -> Result<()>;
```

Comment:
It seems that if one store block in send, others can do nothing but wait too here, and all the transport will be blocked.

Is it ok?

@BusyJay